protocol/minimum-base-fee.md

```diff
@@ -0,0 +1,96 @@
+# Minimum Base Fee
+
+#### Metadata
+
+Authors: @niran
+Created: 2025-07-01
+
+---
+
+# Purpose
+ 
+This document proposes establishing a configurable minimum base fee for OP Stack chains to shorten the length of priority fee auctions.
+
+# Summary
+
+We introduce a new configuration value that can be modified via `SystemConfig`:
+
+| Name             | Type    | Default | Meaning |
+|------------------|---------|---------|---------|
+| `minBaseFeeLog2` | `uint8` | `0`     | The log2 of the minimum base fee that can be charged for a transaction |
```

Comment:
This technically changes the minimum base fee from 0 to 1 wei, although maybe that is an impossible condition in the OP stack as all blocks must contain the attributes transaction.